src/applications/mhv-medical-records/reducers/vitals.js

```diff
@@ -67,20 +67,24 @@ export const getMeasurement = (record, type) => {
 
 export const extractLocation = vital => {
   if (isArrayAndHasItems(vital.performer)) {
-    if (isArrayAndHasItems(vital.performer[0].extension)) {
-      const refId = vital.performer[0].extension[0].valueReference?.reference;
+    const firstPerformer = vital.performer[0];
+
+    if (isArrayAndHasItems(firstPerformer?.extension)) {
+      const refId = firstPerformer.extension[0]?.valueReference?.reference;
       const location = extractContainedResource(vital, refId);
       return location?.name || EMPTY_FIELD;
     }
 
     // Look for Organization references (to handle Lighthouse data)
     const organizations = vital.performer.filter(performer =>
-      performer.reference?.includes('/Organization/'),
+      performer?.reference?.includes('/Organization/'),
     );
+
     if (organizations.length) {
-      return organizations.map(organization => organization.display).join(', ');
+      return organizations.map(org => org?.display || EMPTY_FIELD).join(', ');
```

Comment:
There's no need for `EMPTY_FIELD` here. We are just mapping a list and the list should not have any empty items.